tests/sentry/flags/endpoints/test_hooks.py

```diff
@@ -49,6 +49,33 @@ def test_generic_post_create(self):
             assert response.status_code == 200, response.content
             assert FlagAuditLogModel.objects.count() == 1
 
+    def test_unleash_post_create(self):
+        request_data = {
+            "id": 28,
+            "tags": [{"type": "simple", "value": "testvalue"}],
+            "type": "feature-environment-enabled",
+            "project": "default",
+            "createdAt": "2024-12-30T00:00:00.000Z",
+            "createdBy": "admin",
+            "environment": "development",
+            "createdByUserId": 1,
+        }
+        signature = "testing12345abcdaslkflsldkfkdlks"
+        FlagWebHookSigningSecretModel.objects.create(
+            organization=self.organization,
+            provider="unleash",
+            secret="testing12345abcdaslkflsldkfkdlks",
+        )
+
+        with self.feature(self.features):
+            response = self.client.post(
+                reverse(self.endpoint, args=(self.organization.slug, "unleash")),
+                request_data,
+                headers={"Authorization": signature},
+            )
+            assert response.status_code == 200, response.content
+            assert FlagAuditLogModel.objects.count() == 1
+
```

Comment:
the last assert is failing (`assert FlagAuditLogModel.objects.count() == 0`) but not sure why - need to debug